packages/xod-arduino/platform/program.tpl.cpp

```diff
@@ -72,11 +72,28 @@ TransactionState g_transaction;
 
 #if defined(XOD_DEBUG) || defined(XOD_SIMULATION)
 namespace detail {
-void handleTweaks() {
-    if (XOD_DEBUG_SERIAL.available() > 0 && XOD_DEBUG_SERIAL.find("+XOD:", 5)) {
+void handleDebugProtocolMessags() {
+    {{#forLinkedTetheringInetNode nodes}}
+    bool tetheringInetNodeReceivingData = node_{{ id }}.output_INET->isReceiving();
+    {{/forLinkedTetheringInetNode}}
```

Comment:
Potentially can lead to multiple `tetheringInetNodeReceivingData` definition error. Isn’t it?

Reply:
We squash all tethering nodes into a single one.
So it should not. 🤔